tools/SourceKit/tools/sourcekitd/bin/InProc/sourcekitdInProc.cpp

```diff
@@ -80,31 +80,43 @@ UIdent sourcekitd::UIdentFromSKDUID(sourcekitd_uid_t uid) {
   return UIdent::getFromOpaqueValue(uid);
 }
 
-std::string sourcekitd::getRuntimeLibPath() {
+static llvm::SmallString<128> getToolchainUsrPath() {
```

Comment:
Bit of a nitpick: could we call this "prefix" instead of "usr"?  The fact that we put "usr" in the toolchain prefix by default is platform-specific and probably shouldn't be assumed here.

Reply:
I would also undo the change in return type. SmallString isn’t a great vocabulary type.

Reply:
The `path::append` requires a `SmallVectorImpl<char> &path`.  Typically we would also have `getToolchainPath` take that as an out parameter instead  of a return value, but for a private function I don't know if it matters much.

Reply:
Updated this to change the name, and use an out parameter; I think it makes the intent a little clearer. Thanks for the help with this, I don't know my way around source kit very well yet!